sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleController.java

```diff
@@ -138,14 +140,15 @@ private void createSample(SampleDto sampleDto, Disease disease, Runnable callbac
 	public PathogenTestForm addPathogenTestComponent(CommitDiscardWrapperComponent<? extends AbstractSampleForm> sampleComponent) {
 
 		int caseSampleCount = caseSampleCountOf(sampleComponent.getWrappedComponent().getValue());
-		return addPathogenTestComponent(sampleComponent, null, caseSampleCount, SormasUI::refreshView);
+		return addPathogenTestComponent(sampleComponent, null, caseSampleCount, SormasUI::refreshView, false);
```

Comment:
Why did you set the isDeleteAllowed = false here?
Right now it might work, because the pathogenTest == null, but it think it's a code smell like that.
I my opinion, the isDeleteAllowed should be handed through as a parameter (so it needs to be a parameter of this method as well).